packages/radio-box-group/src/context.ts

```diff
@@ -0,0 +1,19 @@
+import { createContext, ChangeEventHandler, useContext } from 'react';
+import Size from './Size';
+
+export interface RadioBoxGroupContext {
+  value: string | number | undefined;
+  onChange: ChangeEventHandler<HTMLInputElement>;
+  getNextId: () => string;
+  size: Size | undefined;
+  name: string | undefined;
+}
+
+const context = createContext<RadioBoxGroupContext | null>(null);
+
+export const Provider = context.Provider;
+export const Consumer = context.Consumer;
```

Comment:
Is this being used anywhere? I can't find it 👀

Reply:
It's not, I added this export before I checked if `RadioBox` was a function or a class component

Reply:
We can rm since its not :)